security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -383,6 +383,7 @@
                         <dns_yandex>Yandex PDD API</dns_yandex>
                         <dns_zilore>Zilore DNS API</dns_zilore>
                         <dns_zonomi>zonomi.com domain API</dns_zonomi>
+                        <dns_gdnsdk>GratisDNS.dk</dns_gdnsdk>
```

Comment:
alphanumeric sorting would be appreciated :)
I'll fix this in a post-merge commit.